### ISSUE TITLE
Error output  when I try to install naked

### ISSUE THREAD

**Issue by nooperpudd** [chrissimpkins/naked#4]
```
aked/Naked/scratchpad.py ...
  Sorry: IndentationError: expected an indented block (scratchpad.py, line 27)

```

So, U need to fix the problems in the `scratchpad.py` files in line 27

https://github.com/chrissimpkins/naked/blob/master/lib/Naked/scratchpad.py#L27

Please remove the unused commented code. 
Thanks.

**Comment by chrissimpkins** [comment]
Thank you very much!  I will fix it later today and get a new release out.  Strange I have installed it numerous times and never received the error.  What platform are you on?

**Comment by chrissimpkins** [comment]
Fixed.  Thanks much for reporting this.  I just released the new version as v0.1.31 here on the Github repository and pushed it to PyPI.  Let me know if you are still having issues with it. 

-C
